pkg/pull/pull.go

```diff
@@ -277,7 +277,8 @@ func Pull(upstreamURI string, pullOptions PullOptions) (string, error) {
 		IsGKEAutopilot:      k8sutil.IsGKEAutopilot(clientset),
 		IncludeMinio:        pullOptions.IncludeMinio,
 	}
-	if err := upstream.WriteUpstream(u, writeUpstreamOptions); err != nil {
+	err = upstream.WriteUpstream(u, writeUpstreamOptions)
```

Comment:
why is this change necessary? felt cleaner when the `err` was assigned inline

Reply:
my bad, I wasn't looking at the overall diff, just making progressive changes where I modified this line, then later modified it back to "original", but accidentally with a different style